x/lockup/keeper/msg_server.go

```diff
@@ -46,14 +46,7 @@ func (server msgServer) LockTokens(goCtx context.Context, msg *types.MsgLockToke
 			return nil, err
 		}
 
-		ctx.EventManager().EmitEvents(sdk.Events{
-			sdk.NewEvent(
-				types.TypeEvtAddTokensToLock,
-				sdk.NewAttribute(types.AttributePeriodLockID, utils.Uint64ToString(lockID)),
-				sdk.NewAttribute(types.AttributePeriodLockOwner, msg.Owner),
-				sdk.NewAttribute(types.AttributePeriodLockAmount, msg.Coins.String()),
-			),
-		})
+		events.EmitAddTokenToLock(ctx, lockID, msg.Owner, msg.Coins.String())
```

Comment:
I think we should move this inside keeper so that if another module ends up calling `AddToExistingLock`, the event does not get missed